### PR TITLE
Added hover 3d effects

### PR DESCRIPTION
**Pull request by dayyan1341**
Added a scale change of 0.05 on hover as well as moved the box shadow to when hovered

### CODE REVIEW

**Comment by dayyan1341** [comment]
closes #4489 

Normal :- 
image


Hovered :- 
image

**Comment by shiftkey** [comment]
This seems to cause the "last updated" options to stack.

They currently look like this:

Screenshot 2024-07-16 at 8 39 21 AM

But with this PR they now look like this:

Screenshot 2024-07-16 at 8 39 15 AM

I also don't seem to see the same styling effect on the cards themselves:

Screenshot 2024-07-16 at 8 40 19 AM

They do the zoom effect on hover, but there's lots of other regressions here that mean I can't accept this in it's current form.

You can test the changes out by pushing more commits to this branch, which are then published here:

https://deploy-preview-4712--up-for-grabs-test-bench.netlify.app/?pr=4712#/

File: stylesheets/stylesheet.css [hunk 1]
```diff
@@ -778,15 +778,22 @@ form {
 
 .data-box {
   background-color: var(--databox-bg);
-  box-shadow: 0px 5px 10px var(--box-shadow-color);
```
Comment (shiftkey) [inline stylesheets/stylesheet.css:781]: This box seems to be used in many other places, and could be the root cause of all these regressions.

**Comment by dayyan1341** [comment]
image

https://deploy-preview-4712--up-for-grabs-test-bench.netlify.app/?pr=4712#/

**Pull request closed without merging**